vpd-tool/src/vpd_tool.cpp

```diff
@@ -0,0 +1,22 @@
+#include "vpd_tool.hpp"
+
+#include <iostream>
+
+namespace vpd
+{
+int VpdTool::readKeyword(const std::string& i_fruPath,
+                         const std::string& i_recordName,
+                         const std::string& i_keywordName,
+                         const bool i_onHardware,
+                         const std::string& i_fileToSave)
```

Comment:
i_fileToSave is not a default argument here. Please add

Reply:
Default arguments in C++ are only specified once, either in the declaration or in the definition, but not in both.
Otherwise will get a compilation error.